<!-- -->
<launch>
	<arg name="marker_size" default="10" />
    <arg name="debug" default="0" />
	<arg name="max_new_marker_error" default="0.08" />
	<arg name="max_track_error" default="0.2" />
	<arg name="cam_image_topic" default="/ar_usb_cam/image_raw" />
	<arg name="cam_info_topic" default="/ar_usb_cam/camera_info" />	
	<arg name="output_frame" default="/usb_cam_up" />

	<node name="ar_track_alvar" pkg="ar_track_alvar" type="individualMarkersNoKinect" respawn="false" output="screen" args="$(arg marker_size) $(arg max_new_marker_error) $(arg max_track_error) $(arg cam_image_topic) $(arg cam_info_topic) $(arg output_frame) $(arg debug)" />

</launch>
